services-turf/src/main/java/com/mapbox/turf/TurfMisc.java

```diff
@@ -36,7 +37,10 @@ private TurfMisc() {
    * @see <a href="http://turfjs.org/docs/#lineslice">Turf Line slice documentation</a>
    * @since 1.2.0
    */
-  public static LineString lineSlice(Point startPt, Point stopPt, Feature line) {
+  @NonNull
+  public static LineString lineSlice(@NonNull Point startPt, @NonNull Point stopPt,
+                                     @NonNull Feature line) {
+    checkNotNull(line.geometry(), "Feature.geometry() == null");
```

Comment:
Can line.geometry().getType() be null?

Reply:
The Feature geometry can be nullable but if the geometry isn't null, the type will never be null.